src/Compilers/CSharp/Test/Symbol/Symbols/ModuleInitializersTests.Ignored.cs

```diff
@@ -0,0 +1,289 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Symbols
+{
+    public sealed partial class ModuleInitializersTests
+    {
+        [Fact]
+        public void IgnoredOnReturnValue()
+        {
+            string source = @"
+using System.Runtime.CompilerServices;
+
+class C
+{
+    [return: ModuleInitializer]
+    internal static void M()
+    {
+    }
+}
+
+namespace System.Runtime.CompilerServices { class ModuleInitializerAttribute : System.Attribute { } }
+";
+            var verifier = CompileAndVerify(source, parseOptions: s_parseOptions);
+
+            verifier.VerifyMemberInIL("<Module>..cctor", expected: false);
```

Comment:
>verifier.VerifyMemberInIL("<Module>..cctor", expected: false); [](start = 12, length = 62)

I think it would be preferred to use "symbolValidator" and assert the there is no method in metadata. For example how ```ModuleTypeStaticConstructorIsNotEmittedWhenNoMethodIsMarkedWithModuleInitializerAttribute``` unit-test does that.